notebooks/official/workbench/predictive_maintainance/predictive_maintenance_usecase.ipynb

```diff
@@ -1,12 +1,57 @@
 {
```

Comment:
<p>set delete_bucket to False</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/521/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
Done !!